src/cargo/sources/replaced.rs

```diff
@@ -31,7 +31,8 @@ impl<'cfg> Registry for ReplacedSource<'cfg> {
         }).chain_err(|| {
             format!("failed to query replaced source {}",
                     self.to_replace)
-        })
+        })?;
```

Comment:
could be `.map_err(Into::into)` if you don't want to do this two statement thing.

Reply:
Indeed yeah! I'm not necessarily a huge fan of that, but I found the two-statement return wasn't *so* bad.

Reply:
I have no opinion :-)